### PR TITLE
fix: Stop search in local development

### PR DESCRIPTION
**Pull request by xf-**
If you hover the search, the web crashes in development `yarn start/npm run start`. Check if an searchDocs exist and return then 0 indexes are loaded.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -16,27 +16,6 @@ module.exports = function (context, options) {
     getThemePath() {
       return path.resolve(__dirname, './theme');
     },
-    configureWebpack(config) {
-      const generatedFilesDir = config.resolve.alias['@generated']
-      languages = utils.generateLunrClientJS(generatedFilesDir, options.languages);
-      // Ensure that algolia docsearch css is its own chunk
```
Comment (lelouch77) [inline src/index.js:22]: This will break the lanuage support

**Review by lelouch77 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (xf-) [reply src/index.js:22]: Dry I applyed the fix for my repo. Should be oy first commit. I will fix the PR tomorrow

Comment (lelouch77) [reply src/index.js:22]: Sure

File: src/theme/SearchBar/lib/lunar-search.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import lunr from "@generated/lunr.client";
+import * as lunr from "lunr";
```
Comment (lelouch77) [inline src/theme/SearchBar/lib/lunar-search.js:1]: Also please revert this changes too - related to lanuage support

Comment (xf-) [reply src/theme/SearchBar/lib/lunar-search.js:1]: Was not intend, only the first commit. My mistake - i needed those changes at the moment for Webpack 5 support.
